src/content/drafts/introducing-bulk-suppressions.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Introducing bulk suppressions
+teaser: "How to enable stricter linting incrementally"
+tags:
+  - Suppressions
+  - Command Line
+authors:
+  - softius
+categories:
+  - Announcements
+---
+
+At some point, every development team faces the same challenge: you want to enable a new ESLint rule to improve code quality, but your existing codebase already contains numerous violations. When the rule isn't auto-fixable, the path to enabling it as an error can be challenging.
+
+Today, we're excited to introduce ESLint's new violation suppression system, designed to make adopting stricter linting rules more manageable. Let's explore how it works and how it can help your team enhance code quality incrementally.
+
+## Why bulk suppressions?
+
+When enabling a rule in an established project, teams typically choose between fixing all violations upfront, using warnings instead of errors, disabling rules in specific files, or adding inline disable comments. Each approach has merit, but none provides an ideal balance between enforcing standards for new code while addressing existing violations over time.
+
+ESLint's new suppression system offers a more flexible approach by:
+
+1. Recording existing violations in a separate file
+2. Enforcing the rule as an error for all new code
+3. Suppressing recorded violations, allowing you to address existing violations at your own pace
```

Comment:
```suggestion
ESLint's new suppression system offers a more flexible approach, allowing you to:

1. Record existing violations in a separate file that can be checked in to source control.
2. Enable the rule as an error. This will now only apply to new code.
3. Address existing violations recorded in the suppressions file at your own pace.
```